packages/plugins/src/SidebarPlugin.ts

```diff
@@ -40,8 +25,6 @@ function sortByPathLevel(pathA, pathB) {
   return pathBLevel - pathALevel;
 }
 
-const filterPages = page => !(page.sidebar && page.sidebar.exclude);
```

Comment:
we need this code as some pages in Developer have exclude set, to not show them in the sidebar.
This is used when you want to hide pages.

You can create a page and have it as a loadable page, accessed from the full URL but not visible via the menus.